tsconfig/browser.json

```diff
@@ -2,7 +2,7 @@
   "$schema": "https://json.schemastore.org/tsconfig",
   "extends": "./base.json",
   "compilerOptions": {
-    "lib": ["ES2022", "DOM", "DOM.Iterable", "ESNext.Disposable"],
+    "lib": ["ES2023", "DOM", "DOM.Iterable", "ESNext.Disposable"],
```

Comment:
NodeJS 18 (our minimal version) uses ES2023.

The `api` package already relies on ES2023 features (`Array.findLast`).